apps/dashboard/app/models/active_jobs/jobs_json_request_handler.rb

```diff
@@ -85,7 +86,9 @@ def convert_info(info_all, cluster)
           username: j.job_owner,
           extended_available: extended_available,
           nodes: j.allocated_nodes.map{ |node| node.name }.reject(&:blank?),
-          delete_path: users_job?(j.job_owner) ? UrlHelper.instance.delete_job_path(pbsid: j.id, cluster: cluster.id.to_s) : ""
+          delete_path: users_job?(j.job_owner) ? UrlHelper.instance.delete_job_path(pbsid: j.id, cluster: cluster.id.to_s) : "",
+          # Check if Support Ticket is enabled via the support_path definition
+          support_path: users_job?(j.job_owner) && !user_config.support_ticket.empty? ? UrlHelper.instance.support_path(job_id: j.id, cluster: cluster.id.to_s) : ""
```

Comment:
Can we move `support_path` to `config.js` and pull it from there? I think I'm wanting uniformity here.

https://github.com/OSC/ondemand/blob/f25411d7eb591626e0b91b8f49c42b6549170491/apps/dashboard/app/views/layouts/_config.html.erb#L10

and 

https://github.com/OSC/ondemand/blob/f25411d7eb591626e0b91b8f49c42b6549170491/apps/dashboard/app/javascript/config.js#L71

Reply:
ok - will do